.examples/README.md

```diff
@@ -9,15 +9,15 @@ gibt es auch ein Container Image für Docker.
 
 ## docker-compose
 
-### Mit MariaDB
+### Mit PostgreSQL
```

Comment:
are we aiming to drop mariadb / mysql support with release of `v5.x` or is there a coexistence of both?

Reply:
The first idea was to drop mariadb and use postgres. If we can support both in the 5.x that would be an idea and then drop Maria with 6.x